charts/cloudzero-prometheus-agent/templates/secret.yaml

```diff
@@ -1,6 +1,6 @@
-{{- if .Values.credentials.createSecret -}}
-{{- if not .Values.credentials.api_key }}
-{{- fail "api_key must be provided if API key is not provided by an existing secret" }}
+{{- if .Values.createSecret -}}
+{{- if not .Values.apiKey }}
```

Comment:
I think this goes in the `cloudzero` block

Reply:
We removed the cloudzero block